Gio-camacho/week-5/resume-fullstack/frontend/src/app/app.component.html

```diff
@@ -0,0 +1 @@
+<app-cv></app-cv>
```

Comment:
I would have join everything in this component instead of in the cv component, at the end, the app component is empty and useless, but maybe you are going to add extra functionality to it.

Reply:
I have tried to implement SOLID in this application. Although I would be glad that you show me your thinking process. Thanks